.github/workflows/test_code_graph_example.yml

```
@@ -19,5 +19,9 @@ jobs:
       secrets:
         OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
         LLM_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        EMBEDDING_PROVIDER: "fastembed"
```

actionlint (1.7.4)

secret "EMBEDDING_PROVIDER" is not defined in "./.github/workflows/reusable_python_example.yml" reusable workflow. defined secrets are "GRAPHISTRY_PASSWORD", "GRAPHISTRY_USERNAME", "LLM_API_KEY", "OPENAI_API_KEY"

(workflow-call)